exercises/practice/simple-cipher/src/simple_cipher.cr

```diff
@@ -0,0 +1,13 @@
+class SimpleCipher
+  def key
+    raise "Please implement the key method"
```

Comment:
Just one tiny nit, I think writing the stub file like this is nothing wrong with, and is the style for the concept exercises. However, the practice exercises don't use this style but only have a comment and don't use raise. like here: https://github.com/exercism/crystal/blob/main/exercises/practice/atbash-cipher/src/atbash_cipher.cr.

So for consistency, I would like that style to be used:
```
class SimpleCipher
   # Write your code for the 'Simple Cipher' exercise in this file.
  def key
  end

  def encode(text : String)
  end

  def decode(text : String)
  end
end
```